tools/ignored_funcs.txt

```diff
@@ -0,0 +1,100 @@
+
```

Comment:
Please explain this file

Reply:
no u

Reply:
ignored_funcs is a list of functions that should not be added to symbol_addrs, because they're actually bogus functions from the dead code sections

Reply:
makes sense